docs/BindingsGuide_Javascript.md

```diff
@@ -0,0 +1,166 @@
+# Azure SQL bindings for Azure Functions - Javascript
+
+## Input Binding
+
+### function.json Properties for Input Bindings
+
+_TODO_
```

Comment:
What do you mean to put in this section exactly as it is prior to the setup for Input Bindings

Reply:
Same as your powershell examples - it will be a table of the properties that the user should have in their function.json

https://github.com/Azure/azure-functions-sql-extension/blob/main/docs/BindingsGuide_PowerShell.md#functionjson-properties-for-input-bindings